src/core/server/plugins/plugins_service.ts

```diff
@@ -217,6 +218,7 @@ export class PluginsService implements CoreService<PluginsServiceSetup, PluginsS
           if (plugin.includesUiPlugin) {
             this.uiPluginInternalInfo.set(plugin.name, {
               publicTargetDir: Path.resolve(plugin.path, 'target/public'),
+              publicAssetsDir: Path.resolve(plugin.path, 'public/assets'),
```

Comment:
LP exposes the whole folder https://github.com/elastic/kibana/pull/60490/files#diff-15c0a3d71c405cd45e19b0114aecb64eR76